dhis-2/dhis-test-e2e/src/test/resources/tracker/importer/events/events.json

```diff
@@ -117,8 +117,6 @@
           "providedElsewhere": false
         },
         {
-          "updatedAt": "2019-03-04T15:12:59.209",
```

Comment:
I am not sure.
Maybe we can find a way to have a `jsonMapper` just for tests in the integration tests that is going to be strict and fail if there are unknown fields.
In e2e tests that wouldn't work because we are using the view model and those fields are actually used in the output.
Anyway, I cannot think of something easy to implement.